docs/config/proxy.md

```diff
@@ -4,8 +4,9 @@
 |-----------------------|----------------------------------------------------------------------------------------|
 | preserve_hosts        | Enable the [preserve host](/docs/proxy/preserve_host_property.md) definition           |
 | listen_path           | Defines the [endpoint](/docs/proxy/request_uri.md) that will be exposed in Janus       |
-| upstreams             | Defines the OAuth provider's [endpoints](/docs/proxy/upstreams.md) for the request   |
+| upstreams             | Defines the [endpoints](/docs/proxy/upstreams.md) that the request will be forwarded to|
 | strip_path            | Enable the [strip URI](/docs/proxy/strip_uri_property.md) rule on this proxy           |
-| enable_load_balancing | Enable load balancing for this proxy                                                   |
 | methods               | Defines which [methods](/docs/proxy/request_http_method.md) are enabled for this proxy |
 | hosts                 | Defines which [hosts](/docs/proxy/request_http_header.md) are enabled for this proxy   |
+| forwarding_timeouts.dial_timeout | The amount of time to wait until a connection to a backend server can be established. Defaults to 30 seconds. If zero, no timeout exists |
```

Comment:
Can you point here that interval is in parsable `time.Duration` add link to Parse function with examples in go docs? Now the format is not clear here and in the parameter bellow.